os/net/ipv6/sicslowpan.c

```diff
@@ -1464,6 +1462,46 @@ send_packet(linkaddr_t *dest)
   watchdog_periodic();
 }
 /*--------------------------------------------------------------------*/
+/**
+ * \brief This function is called by the 6lowpan code to copy a fragment's
+ * payload from uIP and send it down the stack..
```

Comment:
It's a super tiny thing, but ".." at the end must be a single period.

Reply:
fixed :)